src/main/java/nextstep/optional/Expression.java

```diff
@@ -14,12 +16,11 @@ private static boolean matchExpression(Expression e, String expression) {
     }
 
     static Expression of(String expression) {
-        for (Expression v : values()) {
-            if (matchExpression(v, expression)) {
-                return v;
-            }
-        }
-
-        throw new IllegalArgumentException(String.format("%s는 사칙연산에 해당하지 않는 표현식입니다.", expression));
+        return Arrays.stream(values()).filter(v -> matchExpression(v,
+                expression))
```

Comment:
```suggestion
                .filter(v -> matchExpression(v, expression))
```
요렇게 라인을 나눠보시면 가독성이 조금 더 올라갈 것 같아요!